Changelog.md

```diff
@@ -1,5 +1,9 @@
 # FOSSA CLI Changelog
 
+## v3.8.17
```

Comment:
We generally try to release most new versions to Winget now. I'll send you a Slack with a bit of information.